meson.build

```diff
@@ -276,4 +276,9 @@ if get_option('apps')
 endif
 
 pkg = import('pkgconfig')
-pkg.generate(nanoarrow_lib)
+pkg.generate(libraries : nanoarrow_lib,
+             subdirs : [incdir],
+             version : meson.project_version(),
+             name : 'Apache Arrow nanoarrow',
+             filebase : 'nanoarrow',
```

Comment:
You can remove `filebase` argument - without an argument it will be the same value

Reply:
Right exactly, remove them all.